Warn instead of error when AndroidManifest.xml is missing for Android commands (#264)
bugsnag/bugsnag-cli: CHANGELOG.md

## unreleased

### Changed

> - Changed AndroidManifest.xml handling for Android-related uploads (NDK, Proguard, and React Native Android) to log warnings instead of returning errors when the manifest file cannot be found or read. [#264](https://github.com/bugsnag/bugsnag-cli/pull/264)

Reviewer: ```suggestion
- Changed AndroidManifest.xml handling for Android-related uploads (NDK, Proguard, and React Native Android) to not error when the manifest file cannot be found or read. [#264](https://github.com/bugsnag/bugsnag-cli/pull/264)
```
